service/strong_consistency/sc_groups_manager.cc

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include "sc_groups_manager.hh"
```

Comment:
Why can't we extend existing `raft_group_registry`? Will we have both now and the existing one will managed one group only?

Reply:
`raft_group_registry` is owning all raft groups on the shard + connects them to the RPC handlers.  The `raft_group0` is responsible for creation/deletion of group0 raft group. In the same way `sc_groups_manager` is responsible for creation/deletion tablet raft groups -- but they still are hosted in the `raft_group_registry`. I think it's cleaner to have these three components separated.